content/en/real_user_monitoring/session_replay/browser/privacy_options.md

```diff
@@ -53,21 +53,21 @@ After updating your configuration, you can override elements of your HTML docume
 
 Setting `defaultPrivacyLevel` to `mask` mode masks all HTML text, user input, images, links and [`data-*` attributes][1]. Text on your application is replaced with `X`, rendering the page into a wireframe.
 
-{{< img src="real_user_monitoring/session_replay/mask-mode-fixed.png" alt="Mask mode" style="width:70%;">}}
+{{< img src="real_user_monitoring/session_replay/browser-privacy-mask-all.png" alt="Mask mode" style="width:100%;">}}
```

Comment:
I liked the zoom levels of the old screenshots because it made it easier to see the differences when it came to user input, but if you decided to zoom out on purpose, that's understandable!